### PR TITLE
Convert base62_of_int to pure reason

### PR DESCRIPTION
**Pull request by darekrossman**
I have no idea what Im doing.

### CODE REVIEW

**Comment by darekrossman** [comment]
https://github.com/threepointone/bs-nice/issues/5

File: src/nice.re [hunk 1]
```diff
@@ -982,21 +982,20 @@ let insertRule: string => unit = [%bs.raw
...
+let base62_of_int = int => {
+  let num = ref(int * Js.Math.sign_int(int));
+  let conversion = ref("");
+  let symbols =
+    Js.String.split(
+      "",
+      "0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ"
+    );
+  while (num^ > 0) {
+    conversion := symbols[num^ - 62 * (num^ / 62)] ++ conversion^;
```
Comment (thangngoc89) [inline src/nice.re]: You forgot a Math.floor here

Comment (darekrossman) [reply src/nice.re]: From what I can tell, the returned float is automatically truncated to an int, no?

**Comment by darekrossman** [comment]
per convo https://github.com/threepointone/bs-nice/issues/5

**Comment by threepointone** [comment]
thanks much!

**Pull request merged**